src/Microsoft.Unity.Analyzers.Mpack/Manifest.addin.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ExtensionModel>
+
+	<Extension path="/MonoDevelop/Refactoring/AnalyzerAssemblies">
+		<Assembly file="Microsoft.Unity.Analyzers.dll" />
```

Comment:
This one is an odd one. It shouldn't have the .dll extension.
```suggestion
		<Assembly file="Microsoft.Unity.Analyzers" />
```

Reply:
Consistency at its finest.

Reply:
Marius pls.